external-contributions.md

```diff
@@ -0,0 +1,36 @@
+# External Contributions
+> [!NOTE]
+> _At the Guardian, we are committed to coding in public. A significant portion of our source code is openly accessible on our GitHub repository. This openness sometimes leads to contributions or pull requests (PRs) from individuals outside our organisation._
+>
+> _This guidance may not be suitable for all our projects, and is primarily aimed at unsolicited PRs from people who don’t regularly work with us. Projects which find this guidance most relevant may choose to link to this file via a CONTRIBUTING file in the root folder._
+
+## Guidance for External Contributors
```

Comment:
Perhaps worth including a paragraph explaining that because of various restrictive permissions (e.g. chromatic will not run unless you have a valid token) we will be creating an internal branch mirroring their changes and close their original PR but add them as co author and keep their PR description etc etc ?